config/template.cfg

```diff
@@ -1,43 +1,25 @@
 [GAME]
-# real or sim
 type=sim
-# blue or yellow
-our_color=blue
-their_color=yellow
-#positive or negative
-our_side=negative
-# desired timestamp for the ai
-ai_timestamp=0.05
+our_color = blue
+our_side = negative
+ai_timestamp = 0.05
+play_zone = full
+autonomous_play = false
 
 [COMMUNICATION]
-# serial, sim ou disabled
-# serial when you need to control physical robots through the base-station with nrf
-# sim for simulation with grsim
-# disabled you won't send any robot commands, when you want to test and
-#   play with real vision form cameras without grsim or the base-station.
-type=sim
-
-field_port_file=config/field/sim.cfg
-
-#ui-debug thing
+type = sim
+field_port_file = config/field/sim.cfg
 ui_debug_address = 127.0.0.1
+ui_cmd_sender_port = 14444
```

Comment:
Those value should not be in the config file, since they must stay the same between uidebug and ai of same team color

Reply:
removed